packages/core/test/new-api/call.ts

```diff
@@ -1,10 +1,16 @@
 import { assert } from "chai";
 
 import { defineModule } from "../../src/new-api/define-module";
-import { NamedContractCallFutureImplementation } from "../../src/new-api/internal/module";
+import {
+  AccountRuntimeValueImplementation,
+  ModuleParameterRuntimeValueImplementation,
+  NamedContractCallFutureImplementation,
+} from "../../src/new-api/internal/module";
 import { ModuleConstructor } from "../../src/new-api/internal/module-builder";
 import { FutureType } from "../../src/new-api/types/module";
 
+import { assertInstanceOf } from "./helpers";
```

Comment:
I created this helper that does an `assert.instaceOf(v, Class)` and also lets TS know that after that line `v` has type `Class`.

@zoeyTM I think you'll like it.